babel.config.js

```diff
@@ -0,0 +1,48 @@
+const getPresets = () => {
+  switch (process.env.NODE_ENV) {
+    case 'development':
+      return {
+        targets: { node: 'current' },
+        modules: 'commonjs',
+        useBuiltIns: 'usage',
+        include: ['es7.object.entries'],
+      }
+    case 'rollup':
+      return {
+        targets: {
+          browsers: ['last 2 versions'],
+          node: 'current',
+        },
+        modules: false,
+      }
+    case 'production':
+      return {
+        targets: {
+          browsers: ['last 2 versions'],
+          node: '6',
+        },
+        modules: false,
+        useBuiltIns: 'usage',
+        include: ['es7.object.entries'],
+      }
+    case 'cli':
+      return {
+        targets: { node: '6' },
+        modules: 'commonjs',
+        useBuiltIns: 'usage',
+        include: ['es7.object.entries'],
+      }
+    default:
+      break
+  }
+  return {}
```

Comment:
Just a nitpick: this `return {}` can be in `default` branch instead of `break`.

Reply:
Ahh, good find